phpcs.xml.dist

```diff
@@ -17,6 +17,18 @@
 
     <rule ref="Doctrine"/>
 
+    <rule ref="SlevomatCodingStandard.TypeHints.ParameterTypeHint.MissingNativeTypeHint">
+        <exclude-pattern>*/lib/*</exclude-pattern>
+    </rule>
+
+    <rule ref="SlevomatCodingStandard.TypeHints.ReturnTypeHint.MissingNativeTypeHint">
+        <exclude-pattern>*/lib/*</exclude-pattern>
+    </rule>
+
+    <rule ref="SlevomatCodingStandard.TypeHints.PropertyTypeHint.MissingNativeTypeHint">
+        <exclude-pattern>*/lib/*</exclude-pattern>
+    </rule>
```

Comment:
Maybe I should remove those, `git add --patch` the parts that are backwards compatible, and restore these lines. But that would probably be best in a separate PR for the sake of review.

Reply:
Just tried it, and it's not that complicated to untangle, I'll push it as an extra commit.